File: 344. Reverse String/solution.cpp
class Solution {
public:
  void reverseString(vector<char>& s) {
    int left = 0;

    while (left < (s.size() / 2)){
      char temp = s[left];
      s[left] = s[s.size() - 1 - left];
      s[s.size() - 1 - left] = temp;
      left += 1;
    }
  }
};

File: 344. Reverse String/solution.py
class Solution(object):
  def reverseString(self, s):
    """
    :type s: List[str]
    :rtype: None Do not return anything, modify s in-place instead.
    """
    left = 0
    
    while left < (len(s) / 2):
      temp = s[left]
      s[left] = s[len(s) - 1 - left]
      s[len(s) - 1 - left] = temp
      left += 1
      
    return s